src/axolotl/core/trainer_builder.py

```diff
@@ -927,6 +927,8 @@ def build(self, total_num_steps):
             dpo_trainer_kwargs["loss_type"] = "ipo"
             if self.cfg.dpo_label_smoothing:
                 dpo_trainer_kwargs["label_smoothing"] = self.cfg.dpo_label_smoothing
+        elif self.cfg.rl == "kto":
+            dpo_trainer_kwargs["loss_type"] = "kto"
```

Comment:
we renamed it to `"kto_pair"` as technically with DPO data etc. it is a biased form of the loss